src/containers/ConversionForm/style.css

```diff
@@ -0,0 +1,56 @@
+.container{
+    height: 100vh;
+    width: 100%;
+    background-color: rgba(231, 228, 242, 1);
+    padding-left: 30px;  
+    padding-top: 30px;
+}
+
+.container .loading-icon, .container.loading-text {
+    text-align: center;
+}
+
+.upload {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.upload-container {
+    display: flex;
+    flex-direction: row;
+}
+
+.selector-box {
+    width: 200px;
+    margin-right: 20px;
+}
+
+.container button {
+    margin-right: 20px;
+    min-width: 80px;
+}
+
+.container span svg path {
+    /* this makes the down arrow in the select box visible  */
+    fill: var(--light-theme-btn-default-color);
+}
```

Comment:
thank you, swapped it out, not urgent but curious why you prefer color to fill

Reply:
In general, if an svg is acting as an icon or image, I'd consider it best practice to treat it as a static image or character rather than modifying its internal structure if you have the choice. In this case the path in the svg has the attribute `fill="currentColor"`, which explicitly defers to the element's CSS text color.